h2/src/main/org/h2/util/MathUtils.java

```diff
@@ -270,7 +287,7 @@ public static long secureRandomLong() {
      * @param bytes the target array
      */
     public static void randomBytes(byte[] bytes) {
-        ThreadLocalRandom.current().nextBytes(bytes);
+        sRandom.get().nextBytes(bytes);
```

Comment:
I think that this is a very bad idea. Two thread-local variables are allocated instead of one.